.github/workflows/tag-qa.yml

```diff
@@ -0,0 +1,17 @@
+name: Bump version QA
+on:
+  push:
+    branches:
+      - feature/git_tags_and_versioning
```

Comment:
I don't understand why we have this branch name here. Are we supposed to point this to another branch from time to time? 🤔

Reply:
nice catch @grilix this is why PRs exist :D changing it now

Reply:
Glad to hear, I was getting worried 😅

Reply:
@grilix fixed!